### PR TITLE
Community members pagination

### PR DESCRIPTION
**Pull request by ABee-Tech**
### Tasks

- [x] setup api for pagination on community members page
- [x] Implement pagination component on community members page
- [x] pagination ui implementation
- [x] make pageSize dynamic
- [x] remove search actions, reducers and controllers
- [x] community members fetch through react query

### Links and Curls
http://localhost:3000/community-members/:community

### CODE REVIEW

File: src/screens/communityMembers/CommunityMembers.jsx [hunk 1]
```diff
@@ -6,14 +6,16 @@ import { listUsers, searchUsers } from '../../actions/userAction'
...
 import SearchComponent from '../../components/searchComponent/SearchComponent'
 import DashboardLayout from '../../layout/dashboardLayout/DashboardLayout'
+import Pagination from '../../components/pagination/Pagination'
 import './CommunityMembers.css'
 
 function CommunityMembers ({ history }) {
   const dispatch = useDispatch()
   const [search, setSearch] = useState(null)
-  const { members } = useSelector(state => state.listMember)
+  const resourceList = useSelector(state => state.listMember)
```
Comment (lmmrssa) [inline src/screens/communityMembers/CommunityMembers.jsx:15]: why is this called resourceList?

File: api/src/controllers/communityUserController.js [hunk 2]
```diff
@@ -82,22 +83,38 @@ const followCommunity = async (req, res) => {
 
 const getAllMembers = async (req, res) => {
   try {
-    const data = await db.CommunityUser.findAll(
+    const pageSize = 8
```
Comment (lmmrssa) [inline api/src/controllers/communityUserController.js:86]: make pageSize dynamic with default may be 8 or 9 When screen resize send pageSize based on number of item we show for that screensize * 3 rows

**Review by lmmrssa — CHANGES_REQUESTED** [review]

File: src/screens/communityMembers/CommunityMembers.jsx (hunk 1 above)
Comment (ABee-Tech) [reply src/screens/communityMembers/CommunityMembers.jsx]: Changed into memberList.

File: api/src/controllers/communityUserController.js (hunk 2 above)
Comment (ABee-Tech) [reply api/src/controllers/communityUserController.js]: made pageSize dynamic and implemented in community member page.

**Review by dogi — APPROVED** [review]

**Pull request merged**